components/network/asterisk/asterisk.p5m

```diff
@@ -0,0 +1,1040 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Andrey Sokolov. All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/network/asterisk@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="Asterisk is an all-purpose telephony server"
+set name=info.classification value="org.opensolaris.category.2008:Applications/Internet"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+set name=variant.opensolaris.zone value="global" value="nonglobal"
+
+license asterisk.license license='GPLv2'
+
+dir  path=usr
```

Comment:
Sorry, one more (style) issue - don't deliver dir action if it's not strictly necessary. And this is strictly necessary only when your package delivers empty directory (otherwise file action would be enough) or directory with non-standard permissions.